src/tests/components/DateTime.test.jsx

```diff
@@ -25,7 +25,7 @@ test("Renders DateTime correctly", () => {
   const d = new Date();
   const wordDay = DAYS[d.getDay()];
   expect(container).toHaveTextContent(d.getDate());
-  expect(container).toHaveTextContent(d.getMonth());
+  expect(container).toHaveTextContent(d.getMonth() + 1);
```

Comment:
This expect is not very good when it comes to testing as all it does is just check that the number value of the month exists in the date string.
Would recommend removal

Reply:
Maybe @sannan06 rebased with main instead of main-group2?